website/docs/guides/docs/versioning.mdx

```diff
@@ -145,6 +145,37 @@ versioned_sidebars/version-1.0.0-sidebars.json
 </Tabs>
 ```
 
+:::tip
+
+Versioned sidebar files are, like standard sidebar files plugin root relative - so for the example above your versioned sidebar file may look like:
+
+```js
+{
+  "sidebar": [
+    {
+      "type": "autogenerated",
+      "dirName": "."
```

Comment:
You can have autogenerated versioned sidebars?! Does the versioning CLI produce this now?

Reply:
I don't believe the CLI produces these but they do certainly appear to work.

Reply:
> Does the versioning CLI produce this now?

yes we don't substitute autogenerated items anymore for a while 👍 

![CleanShot 2023-11-10 at 18 47 07@2x](https://github.com/facebook/docusaurus/assets/749374/92356093-bd15-41e4-a73b-819d52f42929)